### PR TITLE
Update to CC8 (cloth-config 8.2.88)

### PR DESCRIPTION
**Pull request by Severnarch**
very basic one line code edit for cc8 (better cc because more mods support it now and not cc7)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-org.gradle.jvmargs  = -Xmx1G
+        org.gradle.jvmargs  = -Xmx1G
```
Comment (unitOrNull) [inline gradle.properties]: why is this line indented

Comment (Severnarch) [reply gradle.properties]: i dunno

Comment (Severnarch) [reply gradle.properties]: probably intellij being intellij

Comment (Severnarch) [reply gradle.properties]: also fixed it in the amend commit below

**Comment by Severnarch** [comment]
I've just decided to archive my repository and just leave it until I get confirmation this gets accepted or denied.

**Comment by Severnarch** [comment]
If confirmation isn't received until 01/03/2023, I'll completely erase my repository and close this PR.

**Comment by homchom** [comment]
Sorry. I meant to respond to this. We talked in the Discord and long story short, it seems CC somewhat misuses semantic versioning, so it'd be better to change the supported versions on our end rather than require 8 instead of 7. For this reason we'll close the PR, but **the next version will still support CC8**.

Thanks anyway and sorry about the wait.

**Pull request closed without merging**